lib/matplotlib/backend_bases.py

```diff
@@ -531,6 +531,12 @@ def draw_image(self, gc, x, y, im):
 
         *im*
             the :class:`matplotlib._image.Image` instance
+
+        *trans*
+            If `option_scale_image` returns `True`, an affine
+            transformation may also be passed to `draw_image`.  It
+            should be applied to the image, before applying the
+            translation of `x` and `y`.
```

Comment:
This docstring is a bit confusing. Are you saying that the user should apply the transform to the image (that doesn't make sense to me). Also, "translation" has a very specific meaning that is a subset of "transformation". Was that intentional?